### ISSUE TITLE
AI doesn't stay after unspoof

### ISSUE THREAD

**Issue by nitol** [leminlimez/Nugget#222]
After unspoof to original, Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.

**Comment by MasterMind-del** [comment]
> Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.

Bro, which device are you using? Mine has been working almost 2 weeks 🤔

**Comment by nitol** [comment]
> > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> 
> 
> 
> Bro, which device are you using? Mine has been working almost 2 weeks 🤔

12 pro max

**Comment by Agustine808** [comment]
> > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> 
> Bro, which device are you using? Mine has been working almost 2 weeks 🤔

thanks for your concern,still its not working i can confirm on iphone 15!

**Comment by MasterMind-del** [comment]
> > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> > 
> > 
> > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> 
> thanks for your concern,still its not working i can confirm on iphone 15!

Are you using 18.2 beta?

**Comment by Agustine808** [comment]
> > > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> > > 
> > > 
> > > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> > 
> > 
> > thanks for your concern,still its not working i can confirm on iphone 15!
> 
> Are you using 18.2 beta?

yes i'm using 18.2 beta 2 also i noticed for some reason image playground app is not occurring!

**Comment by MasterMind-del** [comment]
> > > > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> > > > 
> > > > 
> > > > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> > > 
> > > 
> > > thanks for your concern,still its not working i can confirm on iphone 15!
> > 
> > 
> > Are you using 18.2 beta?
> 
> yes i'm using 18.2 beta 2 also i noticed for some reason image playground app is not occurring.

ohk, as for me my device is 12 mini and apple intelligence is still functioning but as for the playground image its keep showing downloading more than a weeks now. Apart from that everything is working fine

**Comment by nitol** [comment]
> > > > > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> 
> > > > > 
> 
> > > > > 
> 
> > > > > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> 
> > > > 
> 
> > > > 
> 
> > > > thanks for your concern,still its not working i can confirm on iphone 15!
> 
> > > 
> 
> > > 
> 
> > > Are you using 18.2 beta?
> 
> > 
> 
> > yes i'm using 18.2 beta 2 also i noticed for some reason image playground app is not occurring.
> 
> 
> 
> ohk, as for me my device is 12 mini and apple intelligence is still functioning but as for the playground image its keep showing downloading more than a weeks now. Apart from that everything is working fine 

Image playground is not for us, unnecessary installation. My concern is after spoofing it is not staying.

**Comment by Agustine808** [comment]
> > > > > > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > thanks for your concern,still its not working i can confirm on iphone 15!
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > Are you using 18.2 beta?
> > 
> > 
> > > 
> > 
> > 
> > > yes i'm using 18.2 beta 2 also i noticed for some reason image playground app is not occurring.
> > 
> > 
> > ohk, as for me my device is 12 mini and apple intelligence is still functioning but as for the playground image its keep showing downloading more than a weeks now. Apart from that everything is working fine
> 
> Image playground is not for us, unnecessary installation. My concern is after spoofing it is not staying.

that is for sure,but those addons are good too

**Comment by Agustine808** [comment]
> > > > > > Even without touching the AI&SIRI settings the AI vanishes randomly. Sometimes it stays for 1 day, sometimes 2 days ... again less than 1 day...so on.
> > > > > 
> > > > > 
> > > > > Bro, which device are you using? Mine has been working almost 2 weeks 🤔
> > > > 
> > > > 
> > > > thanks for your concern,still its not working i can confirm on iphone 15!
> > > 
> > > 
> > > Are you using 18.2 beta?
> > 
> > 
> > yes i'm using 18.2 beta 2 also i noticed for some reason image playground app is not occurring.
> 
> ohk, as for me my device is 12 mini and apple intelligence is still functioning but as for the playground image its keep showing downloading more than a weeks now. Apart from that everything is working fine

thats good,i wish it will be fixed!

**Comment by nitol** [comment]
Anyone tried unspoofing and still having AI using 2.4.1?
